src/store/entities.js

```diff
@@ -35,7 +35,98 @@ export const defaultState = {
     }
 };
 
-export const defaultColumns = () => ([
+export const newDefaultColumns = () => [
+    {
+        key: 'display_name',
+        sortKey: 'display_name',
+        title: 'Name',
+        renderFunc: TitleColumn
+    },
+    {
+        key: 'groups',
+        sortKey: 'groups',
+        title: 'Groups',
+        props: { width: 10 },
+        renderFunc: () =>
+            <React.Fragment>
+                N/A
+            </React.Fragment>
+    },
+    {
+        key: 'tags',
+        title: 'Tags',
+        props: { width: 10, isStatic: true },
+        // eslint-disable-next-line react/display-name
+        renderFunc: (value, systemId) => (
+            <TagWithDialog count={value.length} systemId={systemId} />
+        )
+    },
+    {
+        key: 'system_profile',
+        sortKey: 'operating_system',
+        dataLabel: 'OS',
+        title: (
+            <Tooltip content={<span>Operating system</span>}>
+                <span>OS</span>
+            </Tooltip>
+        ),
+        // eslint-disable-next-line react/display-name
+        renderFunc: (systemProfile) => (
+            <OperatingSystemFormatter
+                operatingSystem={systemProfile?.operating_system}
+            />
+        ),
+        props: { width: 10 }
+    },
+    {
+        key: 'updated',
+        sortKey: 'updated',
+        title: 'Last seen',
+        // eslint-disable-next-line react/display-name
+        renderFunc: (
+            value,
+            _id,
+            {
+                culled_timestamp: culled,
+                stale_warning_timestamp: staleWarn,
+                stale_timestamp: stale,
+                per_reporter_staleness: perReporterStaleness
+            }
+        ) => {
+            return CullingInformation ? (
+                <CullingInformation
+                    culled={culled}
+                    staleWarning={staleWarn}
+                    stale={stale}
+                    render={({ msg }) => (
+                        <React.Fragment>
+                            <DateFormat
+                                date={value}
+                                extraTitle={
+                                    <React.Fragment>
+                                        <div>{msg}</div>
+                    Last seen:{` `}
+                                    </React.Fragment>
+                                }
+                            />
+                            {verifyCulledInsightsClient(perReporterStaleness) && (
+                                <InsightsDisconnected />
+                            )}
+                        </React.Fragment>
+                    )}
+                >
+                    {' '}
+                    <DateFormat date={value} />{' '}
+                </CullingInformation>
+            ) : (
+                new Date(value).toLocaleString()
+            );
+        },
+        props: { width: 10 }
+    }
+];
+
+export const defaultColumns = () => [
```

Comment:
You don't have to keep two distinct arrays. Instead, try the conditional spread operator:

e.g., `[{ a: 42 }, ...(flag ? [{ test: 'abc' }] : []), { foo: 'bar' }]` for falsey `flag` will result in
```
[
  {
    "a": 42
  },
  {
    "foo": "bar"
  }
]
```
and for truthy:
```
[
  {
    "a": 42
  },
  {
    "test": "abc"
  },
  {
    "foo": "bar"
  }
]
```

Reply:
Fixed that